src/Dog.java

```diff
@@ -0,0 +1,28 @@
+public class Dog extends Animals {
+    private static final int maxRunDistance = 500;
+    private static final int maxSwimDistance = 10;
+
+    public Dog(String name, int age) {
+        super(name, age);
+
```

Comment:
I don't see any counter for dogs, but it was required to implement according to the description.